core/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -132,13 +133,12 @@ public static synchronized TachyonFS get(String masterHost, int masterPort, bool
 
   // All Blocks has been locked.
   private final Map<Long, Set<Integer>> mLockedBlockIds = new HashMap<Long, Set<Integer>>();
+  // Mapping from block id to path of the block locked
+  private final Map<Long, String> mLockedBlockIdToPath = new HashMap<Long, String>();
```

Comment:
if the block is locked, then the server will not delete the block, this records the paths of blocks which are locked. and it is a short cut for lockBlock to get path of the block  when user request to lock a block which is already locked.